package.json

```diff
@@ -35,7 +35,6 @@
     "broccoli-asset-rev": "^3.0.0",
     "ember-cli": "~3.3.0",
     "ember-cli-dependency-checker": "^3.0.0",
-    "ember-cli-eslint": "^4.2.1",
```

Comment:
Note: `eslint-plugin-prettier` doesn't play nicely with `ember-cli-eslint`. I think this is OK though -- running linting in your unit tests isn't really necessary so I think it's a fair trade-off for an easier `prettier` experience.

Reply:
I share this opinion 😄 

https://github.com/ember-cli/rfcs/pull/121